hi-ml-cpath/testhisto/testhisto/datamodules/test_histo_datamodule.py

```diff
@@ -0,0 +1,173 @@
+#  ------------------------------------------------------------------------------------------
+#  Copyright (c) Microsoft Corporation. All rights reserved.
+#  Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+#  ------------------------------------------------------------------------------------------
+import pytest
+import torch
+from pathlib import Path
+from typing import List, Optional
+from unittest.mock import MagicMock, patch
+from torch.utils.data import DataLoader, DistributedSampler, RandomSampler, SequentialSampler
+
+from health_cpath.datamodules.base_module import HistoDataModule
+from health_cpath.datamodules.panda_module import PandaSlidesDataModule, PandaTilesDataModule
+from health_cpath.utils.naming import ModelKey, SlideKey
+from health_ml.utils.common_utils import is_gpu_available
+from testhisto.utils.utils_testhisto import run_distributed
+
+
+no_gpu = not is_gpu_available()
+
+
+def _assert_correct_bag_sizes(datamodule: HistoDataModule, max_bag_size: int, max_bag_size_inf: int,
+                              max_expected_bag_sizes: List[int]) -> None:
```

Comment:
This `max_expected_bag_sizes` argument is a bit confusing. Maybe worth clarifying somehow that this refers to the full true bag sizes (assuming I understood it correctly). Both here and in the test functions that call it (e.g. with named local variables).

Reply:
Good point. wiil do!